config/settings/base.py

```diff
@@ -321,6 +321,6 @@
 MANIFEST_HINTS_URL = os.getenv("CORGI_MANIFEST_HINTS_URL")
 
 DISTGIT_DIR = os.getenv("CORGI_DISTGIT_DIR", "/opt/distgit")
-LOOKASIDE_CACHE_BASE_URL = f"https://{os.getenv('CORGI_TEST_PKGS_HOST')}/repo"
+LOOKASIDE_CACHE_BASE_URL = f"https://{os.getenv('CORGI_LOOKASIDE_CACHE_URL')}/repo"
```

Comment:
makes sense for base settings to not depend on any test machinery

Reply:
@JimFuller-RedHat It's not a dependency on test machinery, it's a naming convention to indicate this variable is used directly in tests (see test_sca.py's test_download_lookaside_sources) and must be set in Gitlab for CI to pass.

Other vars, which don't have CORGI_TEST_ as a prefix, either aren't used directly in tests (but may still be needed for them to pass) or aren't used in tests at all and aren't present in Gitlab CI variables.